docs/docs/versions.md

```diff
@@ -32,7 +32,7 @@ import tableauserverclient as TSC
 
 server = TSC.Server('http://SERVER_URL')
 
-server.version = 2.4
+server.version = 2.6
```

Comment:
put quotes around the version number, per Alex's PR

Reply:
OK. I put quotes around the version number '2.6'  and resolved the merge conflict.